StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/Directory.Build.targets

```diff
@@ -0,0 +1,21 @@
+<Project>
+
+  <Target Name="AfterBuild">
```

Comment:
I'm not a fan of this approach to creating NuGet packages. The approach used before does a much better job of integrating with the build. Among other things, it does the following:

1. If you change assembly versions and then build again, it will remove old packages from the output directory
1. The **Clean** operation will remove the NuGet package created by the current build, if it exists in the output directory
1. If you delete just the NuGet package from the output directory, it knows to recreate it as an artifact of the build